pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/basicpocogkehubfeaturemembership/_generated_object_basicpocogkehubfeaturemembership.golden.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: gkehub.cnrm.cloud.google.com/v1beta1
+kind: GKEHubFeatureMembership
+metadata:
+  annotations:
+    cnrm.cloud.google.com/management-conflict-prevention-policy: none
+    cnrm.cloud.google.com/state-into-spec: absent
```

Comment:
For my own learning, how did you generate these?

It seems subsequent runs are diffing against this line:

```
        - 			"cnrm.cloud.google.com/state-into-spec":                       string("absent"),
```

I have some thoughts about this but would love to know how the golden files were captured before.

Reply:
I didn't turn on `KCC_USE_DIRECT_RECONCILERS=GKEHubFeatureMembership` when generating golden files, I updated the golden files